e2e-tests/src/npg/helpers.js

```diff
@@ -35,17 +35,12 @@ export const verifyPayment = async (noticeCode, fiscalCode) => {
 };
 
 export const acceptCookiePolicy = async () => {
-  const acceptPolicyBtn = '#onetrust-accept-btn-handler';
-  const darkFilterXPath = '/html/body/div[2]/div[1]';
+  const acceptPolicyBtn = '#onetrust-close-btn-container > button';
```

Comment:
please remove comment [here](https://github.com/pagopa/pagopa-checkout-tests/blob/66933a2480b16f33f48010764ed407ea40ae7f52/e2e-tests/src/npg/checkout.npg.integration.test.ts#L36) (this line was removed as a temporary e2e test fix since tests works also without click on the `x` button on onetrust popup)